src/markdown-pages/build-apps/create-an-ab-test-application/catalog-info.mdx

```diff
@@ -38,13 +38,13 @@ cd nru-programmability-course/describe-app/ab-test
 
 <Step>
 
-Create the _catalog_ directory:
+Create the _catalog_ directories:
 
 ```sh
 nr1 create --type catalog
 ```
 
-This creates a _catalog_ directory with template files for inputting custom information about your app.
+This creates a root _catalog_ directory and a _catalog_ directory for each artifact with template files for inputing custom information about your app.
```

Comment:
A couple weeks ago, we replaced the term "artifact" with "Nerdpack item", I think we should probably apply that here, too.

Reply:
I was wondering if that was the case. Thanks for the reminder! I'll update this.

Reply:
Fixed in 4edaa7f